extensions/elasticsearch/7.x/wazuh-template.json

```diff
@@ -416,6 +416,9 @@
               "check.rationale": {
                 "type": "text"
               },
+              "check.condition": {

```

Comment:
This field is inside the `oscap` block, it has no relation to SCA but the integration with OpenSCAP.